src/test/scala/com/github/swagger/scala/converter/ScalaModelTest.scala

```diff
@@ -75,13 +75,17 @@ class ScalaModelTest extends AnyFlatSpec with Matchers {
 
   it should "read a model with vector property" in {
     val schemas = ModelConverters.getInstance().readAll(classOf[ModelWithVector]).asScala
+    schemas.size shouldBe 1
```

Comment:
minor: `schemas should have size 1` -- when this assertion fails the error message includes the toString of schemas -- I'll fix that